selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/CreateAndDeleteProjectsTest.java

```diff
@@ -91,6 +91,7 @@ public void createAndDeleteProjectTest() throws ExecutionException, InterruptedE
     mavenPluginStatusBar.waitClosingInfoPanel();
     explorer.waitFolderDefinedTypeOfFolderByPath(CONSOLE_JAVA_SIMPLE, PROJECT_FOLDER);
     explorer.waitFolderDefinedTypeOfFolderByPath(WEB_JAVA_SPRING, PROJECT_FOLDER);
+    notificationsPopupPanel.waitPopUpPanelsIsClosed();
```

Comment:
please, correct the method name `waitPopUpPanelsIsClosed` > `waitPopupPanelsAreClosed`